crates/db_schema/src/schema_setup/diff_check.rs

```diff
@@ -0,0 +1,40 @@
+#![expect(clippy::expect_used)]
```

Comment:
Also mark this file as `cfg[test]` so it cannot accidentally be used from production code and result in crashes from expect.

Reply:
moved the attribute here